bin/katello-installer

```diff
@@ -2,12 +2,29 @@
 
 require 'rubygems'
 require 'kafo'
+require 'yaml'
 
 CONFIG_FILE = './config/katello-installer.yaml'
+ANSWER_FILE = './config/answers.katello-installer.yaml'
```

Comment:
I suppose this shouldn't change, but it could be gotten from the config file instead so there is one source of truth (and not having to worry about doing another sed replacement).